indexer/database/bridge_transfers.go

```diff
@@ -125,6 +125,7 @@ l1_bridge_deposits.*,
 l1_contract_events.transaction_hash AS l1_transaction_hash,
 l1_transaction_deposits.l2_transaction_hash`)
 
+	// TODO join with l1_tokens and l2_tokens
```

Comment:
I wrote some code to do this but decided to not touch the processor until some outstanding processor code that hasn't landed in develop yet is finished